integrations/cloudbuild/smoke-test.yaml

```
@@ -5,14 +5,20 @@ steps:
           - "-c"
           - |
               git config --global --add safe.directory "*"
-              git submodule update --init --recursive
+              python scripts/checkout_submodules.py --shallow --recursive --platform esp32 
```

```suggestion
              python scripts/checkout_submodules.py --shallow --recursive --platform esp32 nrfconnect silabs linux android
```
nit: `platform` is `nargs='+'` argument so you can init all platforms at once.

---

I tried it at first and it failed for me. I may have used a wrong syntax ... still trying things out.
Generally it seems to me that cloudbuild is throttled on gecko_sdk checkouts and I am trying to find out what to do...

---

Updated